DragaliaAPI/DragaliaAPI/Features/StorySkip/IStorySkipService.cs

```diff
@@ -0,0 +1,10 @@
+namespace DragaliaAPI.Features.Quest;
+
+public interface IStorySkipService
```

Comment:
I would get rid of this interface - I know most services in the codebase have an interface, but I've stopped writing them recently as they don't add much value if you only have one implementation.

Reply:
Ah ok, that sounds great. Yeah most of the larger files had interfaces, so I followed suit.